hazelcast/src/main/java/com/hazelcast/internal/dynamicconfig/ClusterWideConfigurationService.java

```diff
@@ -87,39 +92,31 @@
     static final Map<Class<? extends IdentifiedDataSerializable>, Version> CONFIG_TO_VERSION;
 
     //this is meant to be used as a workaround for buggy equals/hashcode implementations
-    private static final boolean IGNORE_CONFLICTING_CONFIGS_WORKAROUND =
-            getBoolean("hazelcast.dynamicconfig.ignore.conflicts");
+    private static final boolean IGNORE_CONFLICTING_CONFIGS_WORKAROUND = getBoolean("hazelcast.dynamicconfig.ignore.conflicts");
```

Comment:
Maybe `hazelcast.internal.dynamicconfig.ignore.conflicts`.

Reply:
Can we change this? Does it count as public API?

Reply:
Well, strictly speaking it's not public API, it was always meant as an internal kill-switch as workaround for potential bugs with dynamic config like [here](https://github.com/hazelcast/hazelcast/issues/12222#issuecomment-361930182).
However what do we gain by changing the property name? There are lots of other similar internal properties which might be useful in certain not-that-common cases but we don't have `.internal.` as part of their name.

Reply:
I am resolving this then, we can always readjust these names in separate PR if needed. This is out of scope for this PR.